ParameterTrendPercentages/ParameterTrendPercentages_1/ParameterTrendPercentages_1.cs

```diff
@@ -147,19 +149,8 @@ public OnPrepareFetchOutputArgs OnPrepareFetch(OnPrepareFetchInputArgs args)
         if (_parameter == null)
             throw new GenIfException("Invalid parameter ID.");
 
-        DateTime start;
-        DateTime end;
-        if (_start != DateTime.MinValue && _end != DateTime.MinValue)
-        {
-            start = _start;
-            end = _end;
-        }
-        else
-        {
-            var now = DateTime.UtcNow;
-            start = DateTime.UtcNow - TimeSpan.FromHours(24);
-            end = now;
-        }
+        _start = TimeZoneInfo.ConvertTimeFromUtc(_start, TimeZoneInfo.Local);
```

Comment:
Not required, the GetTrendDataMessage allows fetching using UTC by setting the 'DateTimeUTC' boolean property to true (yes, this message is a mess). Same for _end.